regparser/history/annual.py

```diff
@@ -20,6 +21,17 @@
                 "CFR-{year}-title{title}-vol{volume}/xml/"
                 "CFR-{year}-title{title}-vol{volume}-part{part}.xml")
 
+# Matches any of the following:
+#    Parts 200 to 219
+#    Parts 200 to end
+#    Part 52 (§§ 52.1019 to 52.2019)
+PART_SPAN_REGEX = re.compile(
+    r'.*parts? ('
+    r'(?P<span>(?P<start>\d+) to ((?P<end>\d+)|(?P<end_literal>end)).*)'
+    r'|((?P<single_part>\d+) \(.*\))'
+    r')',
```

Comment:
I tried removing it, in vain. For some reason, it doesn't work in Python. Oddly, it works here:
http://pythex.org/?regex=.*parts%3F%20(%3FP%3Cspan%3E(%3FP%3Cstart%3E%5Cd%2B)%20to%20((%3FP%3Cend%3E%5Cd%2B)%7C(%3FP%3Cend_literal%3Eend)).*)%7C((%3FP%3Csingle_part%3E%5Cd%2B)%20%5C(.*%5C))&test_string=Parts%2045%20(foo)%0A&ignorecase=1&multiline=0&dotall=0&verbose=0